azure/Kqlmagic/draft_client.py

```diff
@@ -41,12 +41,30 @@ class DraftClient(object):
     _API_VERSION = "v1"
     _GET_SCHEMA_QUERY = ".show schema"
 
-    def __init__(self, conn_kv: dict, domain: str, data_source: str):
+    _CLOUD_AAD_URLS={
+    "public": "https://api.{0}.io",
+    "mooncake":"https://api.{0}.azure.cn",
+    "fairfax":"https://api.{0}.us",
+    "blackforest":"https://api.{0}.de",
+}
```

Comment:
this assumes that applicationInsights and loganalytics will always have the same pattern. I would like not to assume it. and do a if (https://api.loganalytics.io) then map1, if ((https://api.applicationinsights.io)) then map2 else dont-map leave as is

Reply:
and make the same maps identical for now?